sensor/kubernetes/listener/resources/secrets.go

```diff
@@ -265,78 +261,51 @@ func (s *secretDispatcher) processDockerConfigEvent(secret, oldSecret *v1.Secret
 		}
 	}
 
-	// In Kubernetes, the `default` service account always exists in each namespace (it is recreated upon deletion).
-	// The default service account always contains an API token.
-	// In OpenShift, the default service account also contains credentials for the
-	// OpenShift Container Registry, which is an internal image registry.
-	fromDefaultSA := saName == defaultSA
-	isGlobalPullSecret := secret.GetNamespace() == openshiftConfigNamespace && secret.GetName() == openshiftConfigPullSecret
+	s.processSecretForLocalScanning(secret, action, dockerConfig, saName)
 
 	newIntegrationSet := set.NewStringSet()
 	for registryAddress, dce := range dockerConfig {
 		registryAddr := strings.TrimSpace(registryAddress)
 		if registryAddr != registryAddress {
-			log.Warnf("Spaces have been trimmed from registry adress %q found in secret %s/%s",
+			log.Warnf("Spaces have been trimmed from registry address %q found in secret %s/%s",
```

Comment:
is this more of a debug log instead?

Reply:
I'd like to leave it as a warn to call attention to it, that was introduced initially in https://github.com/stackrox/stackrox/pull/11668, the extra space had caused major issues for Sensor in the past and many hours spent troubleshooting, since we don't log at Debug by default - until this log becomes an issue I would prefer to keep it at warn.